### DISCUSSION TITLE
Possible to create a list of catalogue entries as a sort of index?

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#919] [Get Help]
For a collection catalogue or exhibition catalogue, has anyone created a page with a list of catalogue entries to serve as a sort of index? We would like to make one for our Italian Paintings publication. It could look something like this contents page, except we would want to include the inventory number for each object, along with artist and title. 

https://italian-paintings-live.netlify.app/catalogue/

Is this something that could be automated somehow, or do I need to just do this manually, typing up a list of all objects and linking to the correct entry?

Thanks, in advance, for any advice!

**Comment by Erin-Cecele** [comment]
@geealbers is testing a solution that uses Liquid and the `objects.yaml` file. He'll follow up shortly.

**Comment by salger1** [comment]
I don't know if this is necessarily what you are looking for but since we split our catalogue into 4 volumes plus a supplement we wanted to have a way for people to search the whole catalogue. We made a "series contents" page with the object info (and the accession number as of tomorrow's update). [https://nelson-atkins.org/starr1/series-contents/](https://nelson-atkins.org/starr1/series-contents/) I did it manually with a markdown table and links.

**Reply by zsofiaj** [reply]
Thank you for sharing this, @salger1 ! This is very helpful to see. We'll likely end up making a page like this manually as well. 

The Italian Paintings book I'm working on for the Yale University Art Gallery is 3 volumes, but we have them all on one site (https://italian-paintings-live.netlify.app).

**Comment by cbutcosk** [comment]
One automated approach to index creation is to use a `tags` front matter property on catalogue page files and then use `collections` in a template to pull data from those tagged pages. This would require coding though not a _ton_ of it. So for example in `content/catalogue/1.md`:
```markdown
---
title: Cool Cat. Entry
tags: entries
layout: entry
objects:
  - 1a
---

This is a very cool entry!
```
and in a new file, `content/pub-index.liquid`:
```liquid
---
order: 10
layout: page
title: Index
---

<section class='section quire-page__content'>
  <div class='container'>
    <div class='content'>
      <ul>
        {% for entry in collections.entries %}
          <li><a href='{{ entry.data.canonicalURL }}'>{{ entry.data.title }}</a></li>
        {% endfor %}
      </ul>
    </div>
  </div>
</section>
```

We'll get a page like this:
<img width="810" alt="Screenshot 2024-05-16 at 14 11 21" src="https://github.com/thegetty/quire/assets/12992785/72949000-d228-44f5-895e-937413da3114">

This is more or less what we did on [Exhibiting O’Keeffe’s collections entity index](https://publications.okeeffemuseum.org/exhibiting-okeeffe/data-index/). Though tbh even as a developer in some situations I'd still go @salger1 's route and create a manual index page. Human editors can do things that would be complex to do automated-ly, preserving non-developers' ability to change the page can be an advantage, etc.

**Reply by zsofiaj** [reply]
Thank you for this idea, @cbutcosk ! I wondered if something like this might be possible with minimal coding. I think we'll end up creating an index page manually, but helpful to know this is an option as well.
